shogun-nginx/ssl/private/certificate.crt

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
+MIIDETCCAfkCFA5gAxqxpCw5nDlc2Gh/2ET5SXA5MA0GCSqGSIb3DQEBCwUAMEUx
+CzAJBgNVBAYTAkRFMRMwEQYDVQQIDApTb21lLVN0YXRlMSEwHwYDVQQKDBhJbnRl
+cm5ldCBXaWRnaXRzIFB0eSBMdGQwHhcNMjEwNzIyMTQyODAzWhcNMjIwNzIyMTQy
+ODAzWjBFMQswCQYDVQQGEwJERTETMBEGA1UECAwKU29tZS1TdGF0ZTEhMB8GA1UE
+CgwYSW50ZXJuZXQgV2lkZ2l0cyBQdHkgTHRkMIIBIjANBgkqhkiG9w0BAQEFAAOC
+AQ8AMIIBCgKCAQEAmgGYYrGyhjx//4ICNbsCEPLqmQDBSi+97v1Z/Yxrl4n/L9k0
+b2wcVcH9uo4pX1kiTsHJZQ7zONT6yVqV9xhxo5pVw3eTntMuleg2AlD81D3zhwTn
+YpoxqJn5SmYU7RS4qgzuYFCZweQvloZKJ40OOwWhVbHldwVeXQusi6N7pZZDQXD4
+C4AKsm+/ILTumZSp8oog+Epe5lz+tSKyCywlRKFT6WiceZfGqQ7yhErlMdYdyAdG
+C6vQXe5nu/ONJ73vC/VHalpbMT6sOqYK2AgHo6Op6Ds9kqOGwrhI9cp0ZOfJDGbw
+YC2k4fT3nRgaZMtfmvsOgv1Ki/YDvq0vAHI2lQIDAQABMA0GCSqGSIb3DQEBCwUA
+A4IBAQB3ozLlw63Xb1qfmy4O4fGQifsbwOpHMyFpCgPuMg0ZYJO/lm7fGmkn3Oi1
+vl/eOCriMBW8jM8zcyzu5rMTwbNc8NhC59lfARULvRjR/SfNHthTKv5TDavbcUXE
+T0VmVE/B7nYfBx8oPVAaTKxT04aXOq1U2NksWoi795quLg6fa3fz6qsEYHjbV+/v
+h8OpeYAJzINs07dY5HZoNX6ltrwcLSx43vI+CGMbBJOnoAh7vGJuVLh5kWf1DSKJ
+c8zn7Xz4vyFeA4MYenXaRIGEn46Y74TkkWZHVIhExHR88jtfIQvzirpl3SOJLiWe
+ZtWNDNplxKtWvE3ddjw49w4guOsO
+-----END CERTIFICATE-----
```

Comment:
In addition to the dummy certificate we may want to provide a simple script for recreation (e.g. if timed out).

Reply:
See #15.